app/src/main/java/ch/epfl/qedit/view/answer/TestAnswerFragment.java

```diff
@@ -0,0 +1,31 @@
+package ch.epfl.qedit.view.answer;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import ch.epfl.qedit.R;
+
+/** Only for test purposes */
+public class TestAnswerFragment extends Fragment {
```

Comment:
Is this class actually used ? I couldn't find anywhere in the code. If it isn't used, then it will only reduce our coverage without adding functionality, which is not really what we want :-) .

Reply:
The dispatch in the AnswerFormat when running the test I mentioned above concerning the TestAnswerFormat will return it and then it will be bundled in QuestionFragment and later launched. It does only inflate the xml linked to this TestAnswerFormat and will be removed as soon we have a working second AnswerFormat aside MatrixFormat.

Reply:
Ah ok, I see.